src/components/ha-checkbox.ts

```diff
@@ -0,0 +1,18 @@
+import "@material/mwc-checkbox";
+// tslint:disable-next-line
+const MwcCheckbox = customElements.get("mwc-checkbox");
+
+export class HaCheckbox extends MwcCheckbox {
+  protected firstUpdated() {
+    super.firstUpdated();
+    this.style.setProperty("--mdc-theme-secondary", "var(--primary-color)");
+  }
+}
+
+declare global {
+  interface HTMLElementTagNameMap {
+    "ha-checkbox": HaCheckbox;
+  }
+}
+
+customElements.define("ha-checkbox", HaCheckbox);
```

Comment:
let's use decorator.

Reply:
I tried, but got a bunch of type errors:

> Argument of type 'typeof HaFab' is not assignable to parameter of type 'ClassDescriptor | Constructor<HTMLElement>'.
>   Type 'typeof HaFab' is not assignable to type 'Constructor<HTMLElement>'.
>     Type 'HaFab' is missing the following properties from type 'HTMLElement': accessKey, accessKeyLabel, autocapitalize, dir, and 237 more.

Reply:
Got it